docs/Release-Process.md

```diff
@@ -10,14 +10,11 @@ This document lists the tasks required to create a Kata Release.
 ### Check GitHub Actions
 
 We make use of [GitHub actions](https://github.com/features/actions) in the
-[minor](https://github.com/kata-containers/kata-containers/actions/workflows/release-minor.yaml)
-and
-[major](https://github.com/kata-containers/kata-containers/actions/workflows/release-major.yaml)
-files from the
-`kata-containers/kata-containers` repository to build and upload release
-artifacts.
-
-Those actions are manually triggered and are responsible for generating a new
+[release](https://github.com/kata-containers/kata-containers/actions/workflows/release.yaml)
+file from the `kata-containers/kata-containers` repository to build and upload
+release artifacts.
```

Comment:
It seems this would be a good place to document the manual version bump.

Reply:
Please, do it and add to the PR if possible, @gkurz.
I don't want to block it based on that, and coding time is very reduced on my end.